notNeededPackages.json

```diff
@@ -60,6 +60,10 @@
             "libraryName": "agenda",
             "asOfVersion": "4.1.0"
         },
+        "agent-base": {
+            "libraryName": "agent-base",
+            "asOfVersion": "4.3.0"
```

Comment:
I think this should be version 5.0.0. Though 4.3 has a `.d.ts` file, the `package.json` doesn't have a `types` or `typings` field until version 5.0.0.

Reply:
Done.